cmd/protoc-gen-openapi/main.go

```diff
@@ -35,6 +35,7 @@ func main() {
 		EnumType:        flags.String("enum_type", "integer", `type for enum serialization. Use "string" for string-based serialization`),
 		CircularDepth:   flags.Int("depth", 2, "depth of recursion for circular messages"),
 		DefaultResponse: flags.Bool("default_response", true, `add default response. If "true", automatically adds a default response to operations which use the google.rpc.Status message. Useful if you use envoy or grpc-gateway to transcode as they use this type for their default error responses.`),
+		Integer64Type:   flags.String("integer64_type", "integer", `type for int64/uint64/sint64/fixed64/sfixed64. Use "string" for string format`),
```

Comment:
Just nitpicking.
Would `int64_type` or `64bit_types` be a better name?

Reply:
aha, let @timburks decide

Reply:
Do we need a flag for this? I think the [doc you linked](https://developers.google.com/protocol-buffers/docs/proto3#json) should be the source-of-truth for both us and Envoy/grpc-gateway. I'm comfortable with making this fix our only way of handling this.